test/rules_test.js

```diff
@@ -32,6 +32,12 @@ describe('rules', () => {
         ['x^(--(x + 1))', 'x^(x + 1)']
     ])
 
+    suite('rearrange coefficient', rules.REARRANGE_COEFF, [
+        ['y^3 * 5', '5 y^3'],
+        ['yz * 3', '3 yz'],
+        ['3x^2 * 5', '5 (3 x^2)']
```

Comment:
I see, we should handle all 3, but just find a way to restrict it to two factors.